spark/src/main/scala/geotrellis/spark/io/kryo/KryoRegistrator.scala

```diff
@@ -187,7 +187,8 @@ class KryoRegistrator extends SparkKryoRegistrator {
     kryo.register(classOf[geotrellis.raster.histogram.StreamingHistogram.Delta])
     kryo.register(classOf[geotrellis.raster.histogram.StreamingHistogram.Bucket])
     kryo.register(classOf[geotrellis.raster.density.KernelStamper])
-    kryo.register(classOf[geotrellis.raster.summary.polygonal.MeanResult])
+    kryo.register(classOf[geotrellis.raster.summary.polygonal.visitors.MaxVisitor.TileMaxVisitor])
```

Comment:
Do all of these new classes need to be added to the KryoRegistrator? This would include the concrete visitor implementation classes for:
- Min
- Max
- Sum
- Mean
- StreamingHistogram
For both Tile and MultibandTile.

Reply:
If they are serializable via Java serialization - then nope.